python/modem-firmware-1.3+/nrf_cloud_provision.py

```diff
@@ -409,44 +430,65 @@ def set_mfw_ver_in_shadow(api_key, csv_in, res_out):
             return
         res_file_exists = os.path.exists(result_filepath)
```

Comment:
Just to be clear, because I messed up with Github select, I meant this whole block:

```py
 result_filepath = ''
    if res_out:
        result_filepath = check_file_path(res_out)
        if not result_filepath:
            return
        res_file_exists = os.path.exists(result_filepath)
  ```